templates/editor.mustache

```diff
@@ -267,6 +267,26 @@
 						</tbody>
 					</table>
 
+				</div>
+
+				<div id="project">
+					<h3>EDITOR PROJECT</h3>
+					<p>With this you can save your project and continue with it later.</p>
+					<table class="buttons">
+						<tbody>
+							<tr>
+								<td>
+									<button id="exportProject" value="" />Save Project</button>
+								</td>
+								<td>
+									<button id="importProject" onclick="document.getElementById('importProjectFile').click();">
```

Comment:
This should be in the contribution guidelines, but always avoid inline JavaScript. It's harder to maintain and a security risk, as far as I know.